src/Components/Admin/Councelors/index.js

```diff
@@ -70,7 +70,7 @@ const Councelor = () => {
       />
       <div className={styles.titleAndButton}>
         <h3>Councelors</h3>
```

Comment:
Lets change it here as well
```suggestion
        <h3>Counselors</h3>
```

Reply:
Changed it.